clients/app/src/views/screens/Event/components/EventComponent.tsx

```diff
@@ -21,12 +21,41 @@ import Icon from 'views/components/base/Icon';
 import { Icons } from 'assets/icons';
 import { getReadableNumber } from 'utils/helpers/readableNumber';
 import { IEventComponentProps } from '../types';
+import buildEventMembersStore from 'stores/eventMembers';
 
 const EventComponent: React.FC<IEventComponentProps> = ({ data }) => {
   const { pushTo } = useNavigationWithParams();
   const { colors } = useTheme();
   const [ratio, setRatio] = React.useState(3 / 2);
 
+  const eventMembersStore = React.useMemo(() => buildEventMembersStore(), []);
```

Comment:
Is not it better to move outside of Component?

Reply:
same in PR: https://github.com/ultras-io/ultras/pull/228